src/dialects/db2/query-generator.js

```diff
@@ -820,7 +887,9 @@ export class Db2QueryGenerator extends AbstractQueryGenerator {
   }
 
   dropForeignKeyQuery(tableName, foreignKey) {
-    return _.template('ALTER TABLE <%= table %> DROP <%= key %>', this._templateSettings)({
+    const sql = 'ALTER TABLE <%= table %> DROP FOREIGN KEY <%= key %>;';
+
+    return _.template(sql, this._templateSettings)({
```

Comment:
if you're touching this code, could you replace `_.template` with a regular template string?

Reply:
😄 I debated that but I thought it was left for a reason (SQL injection or something).  Yes I can update it.